### PR TITLE
7280 fix ui for speaker

### PR DESCRIPTION
**Pull request by TanyaGordiychuk**

### CODE REVIEW

File: conference/react-js/src/components/UpdateTalk/UpdateTalk.jsx [hunk 1]
```diff
@@ -214,6 +201,18 @@ class UpdateTalk extends PureComponent {
           onChange={this.addonChange}
         />
 
+        <div className="attach-file_wrapper">
+          <RaisedButton
+            containerElement="label"
+            // className="update-talk__button"
```
Comment (YaroslavPerepelychnyi) [inline conference/react-js/src/components/UpdateTalk/UpdateTalk.jsx]: clean?

File: conference/react-js/src/components/UpdateTalk/update-talk.scss [hunk 2]
```diff
@@ -7,24 +7,30 @@
...
-	}
-
 	&__button {
-		margin-left: 12px;
+		//display: flex;
+		//align-items: center;
+		//
+		//&-item {
+			margin-right: 25px;
+		//}
```
Comment (YaroslavPerepelychnyi) [inline conference/react-js/src/components/UpdateTalk/update-talk.scss]: clean?

File: conference/react-js/src/components/UpdateTalk/update-talk.scss [hunk 3]
```diff
@@ -7,24 +7,30 @@
...
-	&_wrapper {
-		display: flex;
-		justify-content: space-between;
+
+	&__item {
+		margin-right: 20px;
+	}
+
+	&__topic {
+		width: 350px !important;
```
Comment (YaroslavPerepelychnyi) [inline conference/react-js/src/components/UpdateTalk/update-talk.scss:23]: Can we avoid "!important" here?

File: conference/react-js/src/components/UpdateTalk/update-talk.scss [hunk 4]
```diff
@@ -7,24 +7,30 @@
...
+	}
+
+	&__type,
+	&__lang,
+	&__level {
+		width: 165px !important;
```
Comment (YaroslavPerepelychnyi) [inline conference/react-js/src/components/UpdateTalk/update-talk.scss:29]: Can we avoid "!important" here?

File: conference/react-js/src/containers/Talks/talks.scss [hunk 5]
```diff
@@ -16,7 +16,7 @@
 		}
 
 	@media ($mnw: $xmdmx) {
-		height: calc(100vh - 200px);
+		height: calc(100vh - 375px);
```
Comment (YaroslavPerepelychnyi) [inline conference/react-js/src/containers/Talks/talks.scss]: Can we use variable for "375px" ? because you changed 200px => 375px in multiple place

**Review by YaroslavPerepelychnyi — CHANGES_REQUESTED** [review]

File: conference/react-js/src/components/UpdateTalk/update-talk.scss (hunk 3 above)
Comment (TanyaGordiychuk) [reply conference/react-js/src/components/UpdateTalk/update-talk.scss:23]: Unfortunately we can`t  because material-ui styles interrupt this style rules

**Review by YaroslavPerepelychnyi — APPROVED** [review]

**Review by AlexKutkanych — APPROVED** [review]

**Pull request merged**